dune

```diff
@@ -0,0 +1,7 @@
+(library
+ (public_name stb_truetype)
+ (foreign_stubs
+  (language c)
+  (flags :standard -O3 -ffast-math -Werror)
```

Comment:
As a side note: i would highly discourage the use of `-ffast-math` on the account of being a largely unsafe option that makes the resulting binary unportable